src/main.js

```diff
@@ -190,6 +193,21 @@ function configureCommandlineSwitchesSync(cliArgs) {
 				}
 			}
 
+			// Locale
+			else if (argvKey === 'locale') {
+				if ('getSystemLocale' in app) {
+					// Pass in the locale to Electron so that the Windows Control Overlay
+					// is rendered correctly.
+					// The if statement can be removed when Electron officially
+					// adopts the getSystemLocale API.
+					// Ref https://github.com/microsoft/vscode/issues/159813
+					// and https://github.com/electron/electron/pull/35697
+					const localeToUse = (!argvValue || argvValue === 'qps-ploc') ?
```

Comment:
Can you add comment on why we need `qps-ploc` check ?

Reply:
When changing to another language pack, the locale gets saved in argv.json. If we read the pseudolanguage pack value (qps-ploc) and pass it to Chromium, it isn't a recognized locale. qps-ploc is based on English anyway, so I'd rather pass the en locale to Chromium in that case.

Reply:
Can you provide an example when `locale: "qps-ploc"` gets set in argv.json ?

Reply:
@deepak1556 when you install the [Pseudo Language Pack](https://marketplace.visualstudio.com/items?itemName=MS-CEINTL.vscode-language-pack-qps-ploc) and set it as the language using `Configure Display Language`.

Reply:
TIL about `Pseudo Language Pack` extension. Thanks @TylerLeonhardt